x/dispensation/handler_test.go

```diff
@@ -21,14 +21,14 @@ func TestNewHandler(t *testing.T) {
 	err := bank.ValidateInputsOutputs(inputList, outputList)
 	assert.NoError(t, err)
 	for _, in := range inputList {
-		_, err := keeper.GetBankKeeper().AddCoins(ctx, in.Address, in.Coins)
+		err := keeper.GetBankKeeper().AddCoins(ctx, sdk.AccAddress(in.Address), in.Coins)
 		assert.NoError(t, err)
 	}
-	msgAirdrop := types.NewMsgDistribution(sdk.AccAddress{}, "AR1", types.Airdrop, inputList, outputList)
-	res, err := handler(ctx, msgAirdrop)
+	msgAirdrop := types.NewMsgDistribution(sdk.AccAddress{}, "AR1", types.DistributionType_DISTRIBUTION_TYPE_AIRDROP, inputList, outputList)
+	res, err := handler(ctx, &msgAirdrop)
 	require.NoError(t, err)
 	require.NotNil(t, res)
 
 	dr := keeper.GetRecordsForNameAll(ctx, "AR1")
-	assert.Len(t, dr, recipients)
+	assert.Len(t, dr.DistributionRecords, recipients)
```

Comment:
Cant we just use a type alias instead creating a new struct for DistributionRecords.

Reply:
Probably in this case but if it needs to be stored directly in keeper or returned from querier then it needs to exist to implement serialisation.